package.json

```diff
@@ -53,6 +53,8 @@
   },
   "devDependencies": {
     "@proxima-oss/eslint-config": "5.0.0",
+    "@proximacapital/slurpy": "0.0.1-3",
```

Comment:
can we rename `slurpy` to `gulp-base`? I also considered `gulp-template` but this feels more like a base to extend from than a "template"